kubernetes/process_manifest.rb

```diff
@@ -3,6 +3,7 @@
 require "util"
 require "yaml"
 require "fileutils"
+require 'json'
```

Comment:
Oh, that is a really interesting point of view. Sure! The guidelines are here:

https://github.com/SUSE/style-guides/blob/master/Ruby.md